controllers/utils/env.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GetEnv(key string, filenames ...string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	envMap, err := godotenv.Read(filenames...)
+	if err != nil {
+		return ""
```

Comment:
What if it fails to read the file ? The error should be returned here

Reply:
I thought about that but chose not to do it. If we do that, we'll need to handle errors throughout the codebase. It's high investment with no return. Since, we use files for testing only. if there are errors, test will catch it and give us the stacktrace. Production will never run with files so we will never hit that error. WDYT?

Reply:
Alternatively, I can os.Exit when we encounter error reading the file. Since it is an irrecoverable error and requires user to manually fix it, it makes sense to exit immediately.